rest/snippets/Config/LicenseEntity.yml

```diff
@@ -99,4 +99,85 @@ LicenseEntity:
           type: string
           example: https://boox.fi/api/user/waqqas.jabbar%40gmail.com/license/1
 
-
+CreateLicenseEntity:
+  type: object
+  required:
+    - data
+  properties:
+    data:
+      type: object
+      required:
+        - type
+        - attributes
+        - relationships
+        - meta
+      properties:
+        type:
+          type: string
+          example: "License"
+          description: Resource type of the resource to create. Must be "License".
+        attributes:
+          type: object
+          required:
+            - payment_method
+          properties:
+            payment_method:
+              type: string
+              example: "billing"
+              description: License purcahse payment method
```

Comment:
Please update this to match the response object

Reply:
fixed